pkgs/development/python-modules/private-gpt/default.nix

```diff
@@ -1,112 +0,0 @@
-{
-  lib,
-  buildPythonPackage,
-  fetchFromGitHub,
-  fetchurl,
-
-  # build-system
-  poetry-core,
-
-  # dependencies
-  cryptography,
-  docx2txt,
-  fastapi,
-  injector,
-  llama-index-core,
-  llama-index-readers-file,
-  python-multipart,
-  pyyaml,
-  transformers,
-  watchdog,
-
-  # optional-dependencies
-  python,
-  huggingface-hub,
-  gradio,
```

Comment:
Do you think we want to drop some of the dependencies which might where only added for this package?